src/main/java/org/sunbird/workflow/service/impl/WorkflowServiceImpl.java

```diff
@@ -892,4 +900,53 @@ public Response updatePendingRequestsToNewMDO(Map<String, Object> request) {
 		return response;
 	}
 
+	public Response getBulkUpdateStatus(String userAuthToken) {
+		Response response = new Response();
+		try {
+			String userId = accessTokenValidator.fetchUserIdFromAccessToken(userAuthToken);
+			if (StringUtils.isEmpty(userId)) {
+				response.setResponseCode(HttpStatus.BAD_REQUEST);
+				response.put(Constants.ERROR_MESSAGE, "Invalid UserId in the request");
+				return response;
+			}
+			Map<String, Object> propertyMap = new HashMap<>();
+			propertyMap.put(Constants.USER_ID, userId);
+			List<Map<String, Object>> userDetails = cassandraOperation.getRecordsByProperties(
+					Constants.KEYSPACE_SUNBIRD, Constants.USER_TABLE, propertyMap, Arrays.asList(Constants.ROOT_ORG_ID));
+			String rootOrgId = null;
+			if (!userDetails.isEmpty()) {
+				rootOrgId = (String) userDetails.get(0).get(Constants.USER_ROOT_ORG_ID);
+			} else {
+				log.error("Record not found in :" + Constants.USER_TABLE + Constants.DB_TABLE_NAME);
+				throw new ApplicationException("Record doesn't exist");
+			}
+			propertyMap.clear();
+			propertyMap.put(Constants.ROOT_ORG_ID, rootOrgId);
+			List<Map<String, Object>> bulkUploadDetails = cassandraOperation.getRecordsByProperties(
+					Constants.KEYSPACE_SUNBIRD,
+					Constants.USER_BULK_UPLOAD,
+					propertyMap,
+					new ArrayList<>());
+			List<Map<String, Object>> fileDetailsResponse = new ArrayList<>();
+			if (!CollectionUtils.isEmpty(bulkUploadDetails)) {
+				for (Map<String, Object> uploadedFileDetails : bulkUploadDetails) {
+					if ("Bulk Upload By MDO Admin".equalsIgnoreCase((String) uploadedFileDetails.get(Constants.COMMENT))) {
+						fileDetailsResponse.add(uploadedFileDetails);
+					}
+				}
+				response.put(Constants.RESPONSE, fileDetailsResponse);
+			} else {
+				log.error("Record not found in : " + Constants.USER_BULK_UPLOAD + Constants.DB_TABLE_NAME);
+				throw new ApplicationException("Record doesn't exist");
```

Comment:
Should return empty response -- that would be the case if MDO doesn't uploaded any file...

Reply:
Will address this comment in the next PR